apiary.apib

```diff
@@ -231,6 +231,132 @@ Get the current authenticated user.
             + language: `nl` (string)
             + function: `Sales` (string)
 
+# Group Activities
+
+## Calendar events [/events]
+
+### events.list [GET /events.list]
+
+Get a list of calendar events.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `255a988d-d4b6-47eb-a5b8-746c24b5171a`,`e41cf7cd-05cd-4f28-b138-d6d585c30c64` (array[string], optional)
+            + user_id: `5309c8c9-e313-47b0-90ba-6850c3dc3e33` (string, required)
```

Comment:
@jenssegers @regularjack if this is required, is it in fact still a filter then? would it make sense to pass it one level up?

Reply:
I see it as a filter that *could* become optional down the road.

Reply:
Discussed on slack, will be kept as a required filter.

 :ok: by @regularjack on 29/11/2017 14:44:56